### PR TITLE
JBIDE-20059 as developer i would like to be able to start livereload …

### PR DESCRIPTION
**Pull request by dazarov**
…without browsing a view

### CODE REVIEW

File: plugins/org.jboss.tools.livereload.ui/plugin.properties [hunk 1]
```diff
@@ -1,2 +1,3 @@
 open_in_webbrowser_via_proxy=Web Browser via LiveReload Server
-open_in_devicebrowser_via_qrcode=Web Browser on External Device...
\ No newline at end of file
+open_in_devicebrowser_via_qrcode=Web Browser on External Device...
+launch_livereload_server=Create and/or Start LiveReload Server...
```
Comment (maxandersen) [inline plugins/org.jboss.tools.livereload.ui/plugin.properties]: "Start LiveReload server..." should be enough.

File: plugins/org.jboss.tools.livereload.ui/src/org/jboss/tools/livereload/ui/internal/command/LaunchLiveReloadServerCommandHandler.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+import org.eclipse.jface.dialogs.IDialogConstants;
+import org.eclipse.wst.server.core.IServer;
+import org.jboss.tools.livereload.core.internal.server.wst.LiveReloadLaunchConfiguration;
+import org.jboss.tools.livereload.core.internal.util.WSTUtils;
+import org.jboss.tools.livereload.ui.internal.util.Logger;
+
+public class LaunchLiveReloadServerCommandHandler extends AbstractHandler {
+
+	@Override
+	public Object execute(ExecutionEvent event) throws ExecutionException {
```
Comment (maxandersen) [inline plugins/org.jboss.tools.livereload.ui/src/org/jboss/tools/livereload/ui/internal/command/LaunchLiveReloadServerCommandHandler.java]: should take current selection and try open the browser for the selected server module or selected resource (in tree or current file in editor).

**Comment by xcoulon** [comment]
@dazarov thanks for this PR ! 
I'll take it over and complete it to open the browser based on the current selection/active editor.

**Comment by xcoulon** [comment]
Replaced by https://github.com/jbosstools/jbosstools-livereload/pull/104

**Comment by xcoulon** [comment]
Thanks for your help, @dazarov !

**Pull request closed without merging**